mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -650,6 +650,291 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class IndoorPatchPointSample(object):
+    """Indoor point sample within a patch. Modified from `<https://github.com/c
```

Comment:
Add r before """ to use reference in the docstring like [HERE](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/kitti_dataset.py#L19).

Reply:
Fix.